stdlib/2and3/zipimport.pyi

```diff
@@ -1,18 +1,25 @@
 """Stub file for the 'zipimport' module."""
 
-from typing import Optional
+import sys
+from typing import List, Optional, Tuple
 from types import CodeType, ModuleType
+if sys.version_info >= (3, 7):
+    from importlib.abc import ResourceReader
 
 class ZipImportError(ImportError): ...
 
 class zipimporter(object):
     archive: str
     prefix: str
-    def __init__(self, archivepath: str) -> None: ...
-    def find_module(self, fullname: str, path: str = ...) -> Optional[zipimporter]: ...
+    def __init__(self, path: str) -> None: ...
```

Comment:
At least on 3.8, the `path` argument also accepts bytes and PathLike objects.

Less importantly, the docstring for this method is wrong:

```
In [11]: zipimport.zipimporter??                                                                                                                            
Init signature: zipimport.zipimporter(path)
Docstring:     
zipimporter(archivepath) -> zipimporter object
```